src/Compatibility/Microsoft.DotNet.ApiSymbolExtensions/AssemblySymbolLoader.cs

```diff
@@ -256,6 +280,40 @@ public IEnumerable<IAssemblySymbol> LoadMatchingAssemblies(IEnumerable<IAssembly
             return matchingAssemblies;
         }
 
+        /// <inheritdoc />
+        public void LogAllDiagnostics(ILog logger, string? customMessage = null)
```

Comment:
The logger should be an instance variable. This type would benefit from using the logger more extensively. I was surprised to see that isn't yet used by this type. It should be passed through the assembly symbol factory's constructor.

Reply:
I addressed this, but just FYI, it affected the pre-existing IAssemblySymbolLoaderFactory and AssemblySymbolLoaderFactory types. I'll push my updates in a few moments.